Application/Sources/UI/Views/HighlightCell.swift

```diff
@@ -90,11 +91,12 @@ struct HighlightCell: View {
                     ZStack(alignment: .leading) {
                         ImageView(source: imageUrl, contentMode: contentMode)
                         if let highlight {
-                            LinearGradient(colors: [.srgGray16.opacity(0.9), .clear], startPoint: .leading, endPoint: .trailing)
+                            DiagonalGradientView(midwayStop: 0.6)
+                            VerticalGradientView(midwayStop: 0.5)
                             DescriptionView(highlight: highlight)
-                                .padding(.horizontal, 60)
-                                .padding(.vertical, 40)
-                                .frame(width: geometry.size.width * 2 / 3, height: geometry.size.height)
+                                .padding(.horizontal, 30)
+                                .padding(.vertical, 30)
```

Comment:
```suggestion
                                .padding(32)
```

- [Twice as compact](https://www.figma.com/design/YdQrty7qXGmAtUEUdtFtzy?node-id=1020-3313#841729523) should be 32.
- group padding in one line